src/ImageCarouselReact/ImageCarouselReact.xml

```diff
@@ -0,0 +1,211 @@
+                        </enumerationValues> 
+                    </property>
+
+                     <property key="imageClickMicroflow" type="microflow" required="false">
+                        <caption>Call Microflow</caption>
+                        <category>Behavior</category>
+                        <description>Microflow to be triggered when an image is clicked.</description>
+                        <returnType type="Void"/>
+                    </property>
+
+                    <property key="openPage" type="form" required="false">
+                        <caption>Open Page</caption>
+                        <category>Behavior</category>
+                        <description>Opens a page when Image is Clicked</description> 
+                    </property>
+
+                </properties>
+            </property>
+            <property key="captionAttr" type="attribute" entityProperty="imageEntity" required="false">
+                <caption>Caption</caption>
+                <category>Data source</category>
+                <description>This fields provides the caption of a carousel item</description> 
+                <attributeTypes>
+                     <attributeType name="String"/> 
+                </attributeTypes>
+            </property>
+
+            <property key="descriptionAttr" type="attribute" entityProperty="imageEntity" required="false">
+                <caption>Description</caption>
+                <category>Data source</category>
+                <description>This fields provides the Description of a carousel item</description> 
+                <attributeTypes>
+                     <attributeType name="String"/> 
+                </attributeTypes>
+            </property>
+            
+            <property key="controls" type="boolean" defaultValue="true">
+                <caption>Use Image Navigation</caption>
+                <category>Carousel</category>
+                <description>Use the image navigation.</description>
+            </property>
+
+            <property key="indicators" type="boolean" defaultValue="true">
+                <caption>Indicators</caption>
+                <category>Carousel</category>
+            <description>Indicators showing index of image on carousel</description>
+            </property>
+
+            <property key="interval" type="integer" defaultValue="5000">
+                <caption>Interval</caption>
+                <category>Carousel</category>
+                <description>Duration of the transition effect (in ms)</description>
+            </property>
+
+            <property key="pauseOnHover" type="boolean" defaultValue="true">
+                <caption>Pause On Hover</caption>
+                <category>Carousel</category>
+            <description>Pause the Slide on mouse Hover</description>
+            </property>
+
+            <property key="slide" type="boolean" defaultValue="true">
```

Comment:
This option is expected by all user to be enabled, so it could be removed from the configuration.